### ISSUE TITLE
Browser?

### ISSUE THREAD

**Issue by chrismbeckett** [unclechu/node-deep-extend#19]
The title does include *Node*, but I was wondering if this is safe to use in the browser?

**Comment by BlueHayes** [comment]
According to the comments in [this issue](https://github.com/unclechu/node-deep-extend/issues/9), it's not designed for the browser. I couldn't comment whether it works well across browsers though (was wondering the same myself).

**Comment by unclechu** [comment]
@chrismbeckett That's right, it wasn't designed for client-side, it have some node-specific stuff like `Buffer`. May be in the future it will be more compatible with front-end, for v1 release I'm planning to make some API for cloning special objects and move dealing with `Buffer`s and other stuff like this to another sub-module as optional logic. Actually there's some [pull request](https://github.com/unclechu/node-deep-extend/pull/13) that I'm gonna use as prototype. But for now I guess you should use another module on client-side.

**Comment by husa** [comment]
Any chance browser support will be released any time soon?

**Comment by unclechu** [comment]
@husa If I have time for that, but no guarantees.

**Comment by unclechu** [comment]
Relevant PR: https://github.com/unclechu/node-deep-extend/pull/27

**Comment by unclechu** [comment]
@everyonesdesign it will be optional behavior in next major release.
